config/jobs/kubernetes-sigs/kubespray/kubespray-presubmits.yaml

```diff
@@ -0,0 +1,20 @@
+presubmits:
+  kubernetes-sigs/kubespray:
+  - name: pull-kubespray-yamllint
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-kubespray
+      testgrid-tab-name: yamllint
+    branches:
+    - master
+    always_run: false
+    skip_report: true
```

Comment:
just to double check, do you want to only run this job on demand?
`always_run: false` means you have to trigger the job manually with `/test jobname`
`skip_report: true` means that you can only see failures in test-grid.

https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#how-to-configure-new-jobs

Reply:
We want to try it out with a manual trigger first. Inspired by the pattern described in https://github.com/kubernetes/test-infra/blob/ff6c355751962efd5d6b44f0f07567a9be1ea4f9/prow/jobs.md

Reply:
ok, sounds good.